### PR TITLE
temporary fix for sql checker config file keys not found

### PR DESCRIPTION
**Pull request by punchparty93**
since the reconfiguration the sql checker wont get the neccessary keys of its config file

### CODE REVIEW

File: sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala [hunk 1]
```diff
@@ -38,31 +37,18 @@ class SQLTask(val filepath: String, val taskId: String){
...
+  final var JDBCDriver: String = "com.mysql.jdbc.Driver"
   /**
     * Class instance connection
     */
-  private val URL: String = config.getString("sql.connectionUri")
-  private val user: String = config.getString("sql.user")
-  private val password: String = config.getString("sql.password")
+  private val URL: String = "jdbc:mysql://mysql2:3306"
+  private val user: String = "root"
+  private val password: String = "SqfyBWhiFGr7FK60cVR2rel"
```
Comment (Schmidie64) [inline sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala:46]: Should the password be here?

File: sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala [hunk 2]
```diff
@@ -38,31 +37,18 @@ class SQLTask(val filepath: String, val taskId: String){
...
-    val dev_config_file_path = System.getenv("CONFDIR") + "/../docker-config/sqlchecker/application_dev.conf"
-    val prod_config_file_path = "/usr/local/appconfig/application.config"
-
-    var config_factory_path = ""
-    if (Files.exists(Paths.get(prod_config_file_path))) {
-      config_factory_path = prod_config_file_path
-    } else {
-      config_factory_path = dev_config_file_path
-    }
-    config_factory_path
```
Comment (andrej-sajenko) [inline sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala:63]: I suggest an alternative approach: we change the loadFactoryConfigPath method something like this

```scala
private def loadConfig() = {
  val prodConfigFilePath = "/usr/local/appconfig/application.config"
  if (Files.exists(prodConfigFilePath)) {
    val configFile = ConfigFactory.parseFile(new File(prodConfigFilePath))
    ConfigFactory.load(appConfig)
  } else {
    ConfigFactory.load()
  }
}
```

File: sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala [hunk 3]
```diff
@@ -24,9 +23,9 @@ import org.json4s.jackson.JsonMethods._
   * @author Vlad Soykrskyy
   */
 class SQLTask(val filepath: String, val taskId: String){
-  private val appConfig = ConfigFactory.parseFile(new File(loadFactoryConfigPath()))
+  logger.info(filepath)
```
Comment (andrej-sajenko) [inline sql-checker/src/main/scala/de/thm/ii/fbs/SQLTask.scala:26]: If you logg information, try to add some context like `logger.info("SQL task for file: " + filepath)`

**Comment by andrej-sajenko** [comment]
Due to our refactoring this pull request is obsolete.

**Pull request closed without merging**